src/modules/statement/stmt.rs

```diff
@@ -68,7 +69,8 @@ pub enum StatementType {
     Mv(Mv),
     CommandModifier(CommandModifier),
     Comment(Comment),
-    CommentDoc(CommentDoc)
+    CommentDoc(CommentDoc),
+    ConstInit(ConstInit)
```

Comment:
Would be helpful to get in the habit of adding a comma to the last item of multi-line Rust lists (applies to struct fields, function definition parameters, function call arguments, `vec![]` initialisations, etc) when you're making changes anyway.  This it makes subsequent pull requests (marginally) nicer.  If this had been done to `CommentDoc(CommentDoc)` when that was added, you would have one changed line here, not two.